examples/wagemos-cosmoskit-nextjs/src/app/remote/page.tsx

```diff
@@ -0,0 +1,131 @@
+'use client'
+import { stringToAccountId } from '@abstract-money/core'
+import {
+  useRemoteAccountIds,
+  useRemoteHosts,
+  useSignAndBroadcast,
+} from '@abstract-money/react'
+import { useAccounts } from '@abstract-money/react'
+import { useCreateRemoteAccount } from '@abstract-money/react'
+import { useConfig } from '@abstract-money/react'
+import { useExecuteOnRemote } from '@abstract-money/react'
+import { useChain } from '@cosmos-kit/react'
+import { useCallback, useMemo, useState } from 'react'
+import { Button } from '../../components/ui/button'
+import {
+  Select,
+  SelectContent,
+  SelectItem,
+  SelectTrigger,
+  SelectValue,
+} from '../../components/ui/select'
+import { WalletButton } from '../_components/wallet-button'
+
+const CHAIN_NAME = 'neutron'
+export default function RemotePage() {
+  const { data: remoteHosts, isLoading, refetch } = useRemoteHosts(CHAIN_NAME)
+  const [chainInput, setChainInput] = useState(remoteHosts?.[0]?.[0])
+
+  const { address } = useChain(CHAIN_NAME)
+
+  const { data: accounts, status } = useAccounts(
+    {
+      args: {
+        chainName: CHAIN_NAME,
+        owner: address || '',
+      },
+    },
+    {
+      enabled: !!address,
+    },
+  )
+
+  const firstAccount = useMemo(
+    () => (accounts ? accounts[0] : undefined),
+    [accounts],
+  )
+
+  const { mutate: createRemoteAccount, isLoading: isCreating } =
+    useCreateRemoteAccount({
+      accountId: firstAccount,
+    })
+  const { mutate: execRemote, isLoading: isExecuting } = useExecuteOnRemote({
+    accountId: firstAccount,
+  })
+
+  const { data: remoteAccountIds } = useRemoteAccountIds({
+    args: {},
+    accountId: firstAccount,
+  })
+
+  const onCreateClick = useCallback(() => {
+    if (!chainInput) {
+      console.log('no chain input')
+      return undefined
+    }
```

Comment:
since `chainInput` is being the condition that is checked for the `disabled` button state, can safely throw an error here instead.

callback that is returning nothing and doing nothing might cause some issues in future.
```suggestion
    if (!chainInput) {
      throw new Error('chainInput is undefined')
    }
```